backend/penndata/views.py

```diff
@@ -370,14 +370,14 @@ def get_usage(self, room, date, num_samples, group_by, field):
             usage = self.get_usage_on_date(room, curr, field)  # usage for curr
             # incorporate usage safely considering None (no data) values
             usage_aggs = [
-                (self.safe_add(acc[0], val), acc[1] + (1 if val is not None else 0))
-                for acc, val in zip(usage_aggs, usage)
+                (self.safe_add(sum, val), count + (val is not None))
```

Comment:
probably still cleaner to put `1 if val is not None else 0`, equivalent code but makes more sense than adding a boolean

Reply:
+1 to this